File: BRB_Documentation/tags/03_18_2014_BRB_Redesign/BRBWeb_Test/src/main/java/deletethisTestData/MonthsList.java
package deletethisTestData;

import java.util.ArrayList;
import java.util.Iterator;
import java.util.List;

import org.testng.annotations.DataProvider;

public class MonthsList {

	@DataProvider(name = "monthsList", parallel = false)
//	public static Iterator<Object[]> createData() {
	public static Iterator<Object[]> monthsList() {
		List<Object[]> data = new ArrayList<Object[]>();
		data.add(new Object[] { "January" });
		data.add(new Object[] { "February" });
		data.add(new Object[] { "March" });
		data.add(new Object[] { "June" });
		data.add(new Object[] { "July" });
		data.add(new Object[] { "August" });
		data.add(new Object[] { "September" });
		data.add(new Object[] { "October" });
		data.add(new Object[] { "November" });
		data.add(new Object[] { "December" });

		return data.iterator();
	}
	
	@DataProvider(name = "monthsAndDaysList", parallel = false)
	public static Iterator<Object[]> monthsAndDaysList() {
		List<Object[]> data = new ArrayList<Object[]>();
		data.add(new Object[] { "January", "1" });
		data.add(new Object[] { "February", "2" });
		data.add(new Object[] { "March", "3"  });
		data.add(new Object[] { "June", "4"  });
		data.add(new Object[] { "July", "5"  });
		data.add(new Object[] { "August", "6"  });
		data.add(new Object[] { "September", "7"  });
		data.add(new Object[] { "October", "8"  });
		data.add(new Object[] { "November", "9"  });
		data.add(new Object[] { "December", "10"  });

		return data.iterator();
	}
}

File: BRB_Documentation/tags/02_25_2014_BRB_PRC_v1/BRBWeb/WebContent/app/controller/models/MonthsList.js
ensureNamespaceExists();

BRB.MonthsList = function(argTranslator) {
	var logPrefix = '[BRB.MonthsList]::';
	this.translator = argTranslator;
	// values of months are according to javax.xml.datatype.XMLGregorianCalendar
	this.data= [
          { value: null, text: 'monthSelect_Text'},
          { value: '1', text: 'January'},
          { value: '2', text: 'February'},
          { value: '3', text: 'March'},
          { value: '4', text: 'April'},
          { value: '5', text: 'May'},
          { value: '6', text: 'June'},
          { value: '7', text: 'July'},
          { value: '8', text: 'August'},
          { value: '9', text: 'September'},
          { value: '10', text: 'October'},
          { value: '11', text: 'November'},
          { value: '12', text: 'December'}
      ];
    
};
BRB.MonthsList.prototype = new BRB.BaseList();